sealer/cmd/run.go

```diff
@@ -54,6 +54,6 @@ func init() {
 	runCmd.Flags().StringVarP(&runArgs.Pk, "pk", "", cert.GetUserHomeDir()+"/.ssh/id_rsa", "set baremetal server private key")
 	runCmd.Flags().StringVarP(&runArgs.PkPassword, "pk-passwd", "", "", "set baremetal server  private key password")
 	runCmd.Flags().StringVarP(&runArgs.Interface, "interface", "i", "", "set default network interface name")
-	runCmd.Flags().StringVarP(&runArgs.PodCidr, "podcidr", "", "", "set default pod CIDR network. example '192.168.1.0/24'")
-	runCmd.Flags().StringVarP(&runArgs.SvcCidr, "svccidr", "s", "", "set default Service CIDR network. example '10.96.0.0/12'")
+	runCmd.Flags().StringVarP(&runArgs.PodCidr, "podcidr", "", "", "set default pod CIDR network. example '100.64.0.0/10'")
+	runCmd.Flags().StringVarP(&runArgs.SvcCidr, "svccidr", "", "", "set default service CIDR network. example '10.96.0.0/22'")
```

Comment:
The example CIDR may cause an AliCloud user to fail to install.

```
podcidr: 192.168.0.0/16
svccidr: 10.96.0.0/12 
```

Reply:
We need a precheck to prevent pod CIDR confict to the host